### DISCUSSION TITLE
Configure label fields and size

### DISCUSSION THREAD

**Discussion by felixmondelo** [marcusolsson/grafana-gantt-panel#77] [Ideas]
Current label is too big for many scenarios, It would be great if we can decide which fields must appear un the label and size of the label.

**Comment by marcusolsson** [comment]
Could you give an (anonymized) example? How many labels are we talking about?

**Comment by marcusolsson** [comment]
Oh, sorry! I remember [our earlier conversation](https://github.com/marcusolsson/grafana-gantt-panel/discussions/43) now.

**Comment by felixmondelo** [comment]
Just to give more options, current we have:

![image](https://user-images.githubusercontent.com/14011841/135294760-7be1cdce-a9f3-4f58-8915-bc858c3f773b.png)

I only need that the first 4 lines disappear:

![image](https://user-images.githubusercontent.com/14011841/135294874-193de4bc-31e7-4777-8af1-8c9a3a8f3252.png)

It would be great if only contextual data appears on the label:

![image](https://user-images.githubusercontent.com/14011841/135295136-5b6544c6-a41c-4287-93df-609c6dc70608.png)

**Comment by felixmondelo** [comment]
Hi @marcusolsson,

We have modified the code that works for us. We modify the file: 

https://github.com/marcusolsson/grafana-gantt-panel/blob/main/src/GanttChart.tsx

And remove from the const tooltipContent the following lines (320 to 329):

                <div className={styles.tooltip.header}>{label}</div>
                {startTimeValue && (
                  <div className={styles.tooltip.value}>Started at: {startField.display!(startTimeValue).text}</div>
                )}
                {endTimeValue && (
                  <div className={styles.tooltip.value}>Ended at: {endField.display!(endTimeValue).text}</div>
                )}
                <div className={styles.tooltip.faint}>
                  {humanizeDuration((endTimeValue || Date.now()) - startTimeValue, { largest: 2 })}
                </div>

This solution is valid for us, maybe if there is an option in the plugin to enable/disable this headers, can be a good solution. What is your opinion?

**Comment by marcusolsson** [comment]
I think it should be configurable. Not sure what to call the option though.

- Show extended tooltip
- Hide tooltip description
- Show tooltip header
- Any others?

**Comment by felixmondelo** [comment]
Show tootltip header sounds great!

**Comment by felixmondelo** [comment]
Hi @marcusolsson I have developed the solution, I attach the modified files from the Master to modify your code and add the possibility to show/hide the header.
[showTooltipHeader.zip](https://github.com/marcusolsson/grafana-gantt-panel/files/7340529/showTooltipHeader.zip)

**Comment by felixmondelo** [comment]
Hi @marcusolsson,

In my previous comment I sent the code, do you prefer that I merge it? or can you modify it?

Thanks and regards.

**Comment by marcusolsson** [comment]
I've decided to move feature requests to GitHub Discussions. In contrast to bug reports, many feature requests turn into open-ended discussions as it can sometimes be difficult to determine whether a feature is worth implementing. This leads to an increasing number of issues, many which may not be worked on any time soon.

To improve the signal-to-noise ratio among issues, I'm therefore moving all feature requests to the Ideas category under GitHub Discussions. Only when someone's started working on a feature, an actual issue will be created for the feature.

I'll admit that this won't necessarily improve response times, or even guarantee a timely response. Hopefully though, it'll give me a chance to actually clear out issues. Thank you for your patience! 🙏
